hooks/pre-commit

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# pre-commit hook for git which builds the docker image for a given cloud-sdk version.
+# Copy into .git/hooks/  folder
```

Comment:
:) you don't have `set -e` on your script, so actually this script will pass even when `docker build` fails or when `docker run` fails or `docker rmi` fails. :)

Git hook is more difficult to script than just scripting around `update.sh`. I personally don’t care much but I err on the side of following the OSS conventions.